source/high-availability.txt

```diff
@@ -318,18 +318,19 @@ If you encounter resource capacity issues, see :ref:`arch-center-resource-capaci
 Plan Your MongoDB Version Changes and Maintenance
 `````````````````````````````````````````````````
 
-We recommend that you run the latest MongoDB version as it allows you to
-take advantage of new features and provides improved security guarantees
-compared with previous versions.
-
-Ensure that you perform MongoDB major version upgrades far before your
-current version reaches `end of life <https://www.mongodb.com/legal/support-policy/lifecycles>`__. 
-
-You can't downgrade your MongoDB version using the {+atlas-ui+}. Because of this,
-when planning and executing a major version upgrade, we recommend that you
-work directly with MongoDB Professional or Technical Services to help you
```

Comment:
noticed we removed the suggestion contacting Technical Services, will leave up to the team if worth preserving this statement